docs/getting_started/install_ez_platform.md

```diff
@@ -1,12 +1,200 @@
+Before executing it make sure that the database user has sufficient permissions.
+
+If Composer asks for your token, you must log in to your GitHub account and generate a new token
+(edit your profile, go to Developer settings > Personal access tokens and Generate new token with default settings).
+This operation is performed only once when you install eZ Platform Enterprise Edition for the first time.
+
+## Use PHP's built-in server
+
+At this point you can use PHP's built-in server: `php bin/console server:start`.
+
+If you want to use an Apache web server, you need to [set up directory permissions](#set-up-permissions) and [prepare a virtual host](#set-up-virtual-host).
+
+## Prepare installation for production
+
+Installation for production is only supported on Linux.
+You should use Mac OS or Windows for development only.
+
+To use eZ Platform with an HTTP server, you need to [set up directory permissions](#set-up-permissions) and [prepare a virtual host](#set-up-virtual-host).
+
+### Set up permissions
+
+The web user must be the owner of all your files (for example with the `www-data` web user):
+
+``` bash
+chown -R www-data:www-data <your installation directory>
+```
+
+Directories `var` and `web/var` need to be writable by CLI and web server user.
+Future files and directories created by these two users will need to inherit those permissions.
+
+!!! caution
+
+    For security reasons, in production there is no need for web server to have write permissions to other directories.
+
+To set up permissions for production, it is recommended to use an ACL (Access Control List).
+See [Setting up or Fixing File Permissions](http://symfony.com/doc/3.4/setup/file_permissions.html) in Symfony documentation
+for information on how to do it on different systems.
+
+### Set up virtual host
+
+#### Option A: Scripted configuration
+
+Use the included shell script: `/<your installation directory>/bin/vhost.sh` to generate a ready to use `.conf` file.
+Check out the source of `vhost.sh` to see the options provided.
+
+#### Option B: Manual configuration
+
+Copy `/<your installation directory>/doc/apache2/vhost.template` to `/etc/apache2/sites-available` as a `.conf` file.
+
+Modify it to fit your project.
+
+Specify `/<your installation directory>/web` as the `DocumentRoot` and `Directory`.
+Uncomment the line that starts with `#if [SYMFONY_ENV]` and set the value like this:
+
+``` bash
+SetEnvIf Request_URI ".*" SYMFONY_ENV=prod
+```
+
+#### Enable virtual host
+
+When the virtual host file is ready, enable the virtual host and disable the default:
+
+``` bash
+a2ensite ezplatform
+a2dissite 000-default.conf
+```
+
+Finally, restart the Apache server. The command may vary depending on your Linux distribution. For example of Ubuntu use:
+
+``` bash
+service apache2 restart
+```
+
+Open your project in the browser and you should see the welcome page.
+
+!!! tip "eZ Launchpad for quick deployment"
```

Comment:
maybe we should move it to the top, where user has a greater chance of reading it before following our installation

Reply:
The problem is, launchpad is not quite the recommended method of installing (yet), especially for production, so I removed it from the top no to get users down a blind alley.